crates/onnx-ir/src/node/eye_like.rs

```diff
@@ -0,0 +1,63 @@
+use crate::ir::{ArgType, Node, TensorType};
+
+/// Create an EyeLike configuration from the node
+/// EyeLike creates an identity matrix with the same shape as the input tensor
+pub fn eye_like_config(_node: &Node) {
+    // EyeLike operation has no configuration parameters
+    // It simply creates an identity matrix with the same shape as the input
```

Comment:
Actually, EyeLike op has configurations, please see: https://onnx.ai/onnx/operators/onnx__EyeLike.html#attributes

BTW, there should be an existing op that maps dtype to Burn's onnx type. 

k attribute reminds me of the offset in diag mask https://burn.dev/docs/burn/tensor/struct.Tensor.html#method.diag_mask

Probably you can use the result (bool) into int using `int()`